app/server.py

```diff
@@ -13,6 +13,8 @@
 
 @app.route('/')
 def index():
+    """this is done because in mongodb the database is not created until and unless something is added to it."""
```

Comment:
That means on every request, we are making a mongodb insert query.

This will definitely increase the response time.

Can you look for other alternatives?